packages/eslint-plugin-pf-codemods/lib/rules/tab-title-text.js

```diff
@@ -0,0 +1,65 @@
+const { getPackageImports } = require('../helpers');
+
+// https://github.com/patternfly/patternfly-react/pull/4146
+module.exports = {
+  create: function(context) {
+    const tabImport = getPackageImports(context, '@patternfly/react-core')
+      .filter(specifier => specifier.imported.name === 'Tab');
+    
+    const addImportIfNeeded = fixer => {
+      const tabTitleTextImport = getPackageImports(context, '@patternfly/react-core')
+        .filter(specifier => specifier.imported.name === 'TabTitleText');
+      if (tabTitleTextImport.length) {
+        return [];
+      }
+      const pfImports = context.getSourceCode().ast.body
+        .filter(node => node.type === 'ImportDeclaration' && node.source.value === '@patternfly/react-core');
+      return fixer.insertTextAfter(pfImports[pfImports.length - 1].specifiers[pfImports[pfImports.length - 1].specifiers.length - 1], ', TabTitleText');
+    }
+
+    return tabImport.length ? {
+      JSXOpeningElement(node) {
```

Comment:
Can we not target `ImportDeclaration`s here? I'm going to check this out locally, but I think that'd be pretty nice...